src/Views/Movie/Index.cshtml

```diff
@@ -0,0 +1,97 @@
+﻿@using Microsoft.AspNetCore.Mvc.Localization

```

Comment:
What is this page?
It's the same as the default `Index.cshtml`, why did you decide to include it here?